customs.js

```diff
@@ -55,6 +57,13 @@
       }
     )
     .then(
+      (function (result) {
+        if (result.lockout) {
+          return this.db.lockAccount(account)
```

Comment:
+1 regex just to keep things simple.

How will we measure the success of this experiment?  I kinda assume none of us are going to trigger lockout unless we go out of our way to do so, for testing purposes.  What criteria will we use to decide when we should enable it more broadly?

(Also on that topic, maybe we should fire a `log.warn` or something here so we can easily measure how often lockout is triggered)